meta-arm-bsp/recipes-bsp/images/core-image-base-uefi-secureboot.inc

```diff
@@ -0,0 +1,20 @@
+# Detected by passing kernel parameter
+QB_KERNEL_ROOT = ""
+
+# kernel is in the image, should not be loaded separately
+QB_DEFAULT_KERNEL = "none"
+
+KERNEL_IMAGETYPE = "Image"
+
+WKS_FILE = "efi-disk-no-swap.wks.in"
+
+MACHINE_FEATURES += "efi"
+
+EFI_PROVIDER = "grub-efi"
```

Comment:
If the plan is to use systemd, e.g. DISTRO_FEATURS += "systemd" then this could/should be systemd-boot. That's what poky does already if systemd is the init manager. This would mean that any grub changes could be dropped.

Reply:
I have replaced grub with systemd-boot. Thanks!